content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -419,6 +419,97 @@ Note that in addition to file output plugin, logstash has a variety of outputs t
 let users route data where they want. For example, users can emit audit events to elasticsearch
 plugin which supports full-text search and analytics.
 
+## Webhook Collector Examples
+
+### Use Falco to collect audit events
+
+[Falco](falco_website) is an open source project for intrusion and abnormality detection for Cloud Native platforms.
+In this example, we will see how to send audit events to the Kubernetes Audit endpoint exposed by Falco.
+
+After receiving Audit Events, Falco applies a set of rules, described below, in order to detect suspicious or notable behavior automatically.
+
+#### Install Falco
+
+You can install Falco standalone or as a Kubernetes DaemonSet following the instructions [here](falco_installation).
+You can also install Falco using the official [Helm Chart](falco_helm_chart).
+
+Once Falco is installed you need to make sure it is configured to expose the Audit webhook.
+
+This is the configuration to enable it.
+
+```yaml
+webserver:
+   enabled: true
+   listen_port: 8765
+   k8s_audit_endpoint: /k8s_audit
+   ssl_enabled: false
+   ssl_certificate: /etc/falco/falco.pem
+```
+
+This configuration is usually in `/etc/falco/falco.yaml`. When installed as a Kubernetes DaemonSet you can edit that file changing the `falco-config` ConfigMap.
+
+#### Configure Kubernetes Audit
+
+1. Create a [kubeconfig file](/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/) for the [kube-apiserver][kube-apiserver] webhook audit backend
+
+        cat <<EOF > /etc/kubernetes/audit-webhook-kubeconfig
+        apiVersion: v1
+        kind: Config
+        clusters:
+        - cluster:
+            server: http://<ip_of_falco>:8765/k8s_audit
+          name: falco
+        contexts:
+        - context:
+            cluster: falco
+            user: ""
+          name: default-context
+        current-context: default-context
+        preferences: {}
+        users: []
+        EOF
+
+2. Start [kube-apiserver][kube-apiserver] with the following options:
+
+    ```shell
+    --audit-policy-file=/etc/kubernetes/audit-policy.yaml --audit-webhook-config-file=/etc/kubernetes/audit-webhook-kubeconfig
+    ```
+
+#### Audit Rules
+
+Rules devoted to Kubernetes Audit Events can be found in [k8s_audit_rules.yaml][falco_k8s_audit_rules]. When installed as a native package or using the official Docker images, Falco installs this rules file to `/etc/falco/`, so they are available for use.
```

Comment:
Hey Tim, thanks a lot for your informal review! Really appreciated :)

Since the Falco rules file can change I don't think atm that copying it here reveals to be a good approach for readers.